### PR TITLE
[14.0] [MIG] l10n_ch_account_tags: Migration to 14.0

### PR DESCRIPTION
**Pull request by sonhd91**

### CODE REVIEW

File: l10n_ch_account_tags/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+The migration of this module from 13.0 to 14.0 was financially supported by Camptocamp
```
Comment (TDu) [inline l10n_ch_account_tags/readme/DESCRIPTION.rst]: I don't think this should be in the `DECRIPTION.rst` file.
Maybe `CREDITS.rst` ?

**Review by TDu — COMMENTED** [review]
Also I don't think we need to new commit for the migration.

File: l10n_ch_account_tags/readme/DESCRIPTION.rst (hunk 1 above)
Comment (sonhd91) [reply l10n_ch_account_tags/readme/DESCRIPTION.rst]: I got it, thanks you !

**Comment by sonhd91** [comment]
The commit [[MIG] add commit from 14-fix-base-bank-test...](https://github.com/OCA/l10n-switzerland/pull/615/commits/d1bf3b4e0f245f4d54b60f61ad9c80ec1ba1f093) will be remove after [PR#616](https://github.com/OCA/l10n-switzerland/pull/616) merged

**Review by mmequignon — APPROVED** [review]
Waiting for this one to be merged https://github.com/OCA/l10n-switzerland/pull/616#pullrequestreview-753978757
I pre-approve.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 exclude: |
   (?x)
   # NOT INSTALLABLE ADDONS
+  ^l10n_ch_base_bank/|
```
Comment (simahawk) [inline .pre-commit-config.yaml]: this is wrong I think

**Review by simahawk — COMMENTED** [review]
LG but please fix .pre-commit file and drop the last commit as #616 is gettting merged

File: .pre-commit-config.yaml (hunk 2 above)
Comment (sonhd91) [reply .pre-commit-config.yaml]: Thanks you, I will fix this soon

**Review by TDu — APPROVED** [review]

**Pull request merged**